ref/general/hardening-intro.adoc

```diff
@@ -37,26 +42,29 @@ To learn more about OS vulnerabilities and ways to prevent OS intrusions, go to
 
 [#network]
 == Network
-Network intrusions occur when unauthorized users monitor or alter network traffic. Replay attacks and man-in-the-middle (MITM) attacks are two examples of types of network intrusions.
+Network intrusions occur when unauthorized users monitor or alter network traffic. Replay attacks and man-in-the-middle (MITM) attacks are two examples of network intrusions.
+// Any other information about network intrusions?
 
 To learn more about network vulnerabilities and ways to prevent network intrusions, go to <link pending>.
 
-==== Best practices
+=== Best practices
 
 * *Use a firewall*: Use a firewall to shield production Open Liberty servers so that they don't run in the DMZ. The DMZ is an unsecured network segment that exists between internal and external networks. Firewalls help prevent security intrusions by monitoring network traffic and blocking specific traffic based on security rules.
 * *Enable the Transport Security feature*: The link:/docs/ref/feature/#transportSecurity.html[Transport Security feature] enables the link:/docs/ref/feature/#ssl.html[SSL feature] to support SSL connections and adds the ability to configure outbound transports. Additionally, disable non-SSL ports by setting them to `-1` in the `server.xml` file, and use only certificates that are signed by a certificate authority for public-facing production servers. Also, enable SSL in your client and web server. Using SSL connections throughout your network communication flow helps protect your server from network intrusions.
 +
-If you use SSO for user authentication, all SSO endpoint URLs must use an HTTPS scheme, and the client must perform a TLS server certificate check.
-* *Run applications on a virtual host*: Configure applications to run on a virtual host. Open Liberty admin traffic comes in on the default host, so configuring your applications to run on a virtual host keeps Open Liberty admin traffic separate from application requests.
+If you use SSO for user authentication, all SSO endpoint URLs must use an HTTPS scheme, and the client must perform an SSL server certificate check.
```

Comment:
In this sentence "If you use SSO for user authentication, all SSO endpoint URLs must use an HTTPS scheme, and the client must perform an SSL server certificate check." avoid the use of the word 'all'. You can say "the SSO endpoint URLs must use an HTTPS scheme,". Check with the SME for technical accuracy.

Reply:
I think I'll keep all and check with the SME on this. I think his point was to stress that all SSO endpoint URLs must use it, but I can see if he's okay with me taking it out.

Reply:
Okay.